app/controllers/application_controller.rb

```diff
@@ -54,7 +54,11 @@ def handle_api_request
 
   def get_user_collections
     if can? :manage, Admin::Collection
-      Admin::Collection.all
```

Comment:
If you do this, you get all collections (even collections you are not the manager of) on the index screen.